tests/pytest/config/default.yml

```diff
@@ -1,5 +1,146 @@
 data_table: default
-diag_table: default
+diag_table:
+  base_time: 2000-01-01 00:00:00
+  name: default
+  file_configs:
+  - field_configs:
+    - {module_name: dynamics, field_name: pk, output_name: pk}
+    - {module_name: dynamics, field_name: bk, output_name: bk}
+    - {module_name: dynamics, field_name: hyam, output_name: hyam}
+    - {module_name: dynamics, field_name: hybm, output_name: hybm}
+    - {module_name: dynamics, field_name: zsurf, output_name: HGTsfc}
+    name: atmos_static
+    frequency: -1
+    frequency_units: hours
+    time_axis_name: time
+    time_axis_units: hours
+  - field_configs:
+    - {module_name: dynamics, field_name: grid_lont, output_name: lon}
+    - {module_name: dynamics, field_name: grid_latt, output_name: lat}
+    - {module_name: dynamics, field_name: grid_lon, output_name: lonb}
+    - {module_name: dynamics, field_name: grid_lat, output_name: latb}
+    - {module_name: dynamics, field_name: area, output_name: area}
+    - {module_name: dynamics, field_name: us, output_name: UGRDlowest}
+    - {module_name: dynamics, field_name: u850, output_name: UGRD850}
+    - {module_name: dynamics, field_name: u500, output_name: UGRD500}
+    - {module_name: dynamics, field_name: u250, output_name: UGRD250}
```

Comment:
Note this diag_table is copied from the [`diag_table_default` in `fv3config`](https://github.com/ai2cm/fv3config/blob/master/fv3config/data/diag_table/diag_table_default).  Something a little odd that I preserved for the sake of not needing to modify the reference checksums was that the default `diag_table` there outputs `u250` and `v200`. I don't know how useful the default `diag_table` is in practice, but we might consider harmonizing the pressure levels used there.